README.md

```diff
@@ -199,11 +197,32 @@ file inside your source code repository.
 
       * `web-clustering`: Support for Infinispan-based web session clustering.
 
+      * All layers defined by [wildfly-extras Galleon feature-pack](https://github.com/wildfly-extras/wildfly-datasources-galleon-pack).
+
+    * All [WildFly Galleon Layers](https://docs.wildfly.org/23/Galleon_Guide.html#wildfly_layers).
+
+* `GALLEON_PROVISION_FEATURE_PACKS`
+
+     A comma separated list of additional Galleon feature-packs identified by Maven coordinates (`<groupId>:<artifactId>:<version>`). These feature-packs must have been 
+     designed to work with WildFly. Galleon layers defined by these feature-packs can be set in `GALLEON_PROVISION_LAYERS` env variable.
```

Comment:
Should we say that these feature must have been designed to work with the wildlfy-feature-pack that is provisioned .
(what would happen if I specify an extra feature-pack depending on WildFly 22.0 feature-pack to build it against WildFLy 23.0 image?)

Reply:
Thanks, good point. I clarified.